### PR TITLE
TIG-1577 update to latest poplar report schema

### PR DESCRIPTION
**Pull request by guoyr**
Depends on https://github.com/10gen/dsi/pull/346

### CODE REVIEW

File: src/python/genny/cedar_report.py [hunk 1]
```diff
@@ -105,7 +106,7 @@ def build_report(config):
             tags=[],
             local_path=path,
             created_at=config.created_at,
-            is_uncompressed=True
+            convert_bson_to_ftdc=True
```
Comment (rtimmons) [inline src/python/genny/cedar_report.py]: personal vendetta against booleans - can this be a `conversion_mode` or something that's an enum e.g. `bson_to_ftdc` or `no_conversion`?

(I realize it was already a boolean and this just gives it a better name so feel free to punt on this)

File: src/python/genny/cedar_report.py [hunk 2]
```diff
@@ -133,6 +134,7 @@ def build_report(config):
         api_token=None,
         region=config.cloud_region,
         name=config.cloud_bucket,
+        permission='public-read',
```
Comment (rtimmons) [inline src/python/genny/cedar_report.py]: can it...not be? I understand it's not sensitive information but usually I see "public-read" and think the permissions are just poorly-defined and we're resorting to ¯\\_(ツ)\_/¯

File: src/python/genny/cedar_report.py (hunk 1 above)
Comment (guoyr) [reply src/python/genny/cedar_report.py]: I think you'd want to have poplar do this. The format of the `namedtuple`s in this file just mirror the schema defined in poplar, it seems weird to make an exception for this one field.

File: src/python/genny/cedar_report.py (hunk 2 above)
Comment (guoyr) [reply src/python/genny/cedar_report.py]: Sadly this is something poplar should do as well :(

I filed https://jira.mongodb.org/browse/MAKE-717 to have the bucket policy be set to disallow public reads.

**Pull request merged**